src/Orchard.ContentManagement.Abstractions/ContentExtensions.cs

```diff
@@ -108,5 +108,25 @@ public static bool HasDraft(this IContent content)
                    content.ContentItem.Published == false ||
                    (content.ContentItem.Published && content.ContentItem.Latest == false);
         }
+
+        public static bool IsLatest(this IContent content)
+        {
+            return content.ContentItem != null && content.ContentItem.Latest;
+        }
+
+        public static bool IsNew(this IContent content)
```

Comment:
Can we remove all the IsNewXXX or are they used? It just adds some confusion, I have no idea how different they are.